modules/social_features/social_follow_user/config/install/views.view.followers_user.yml

```diff
@@ -0,0 +1,336 @@
+langcode: en
```

Comment:
Create the same file in the "static" directory and use a created file in the update hook for installing configuration.